demos/common/src/main/java/dk/alexandra/fresco/demo/cli/CmdLineProtocolSuite.java

```diff
@@ -229,8 +226,7 @@ private Drbg getDrbg(int myId, int prgSeedLength) {
     Map<Integer, RotList> seedOts = new HashMap<>();
     for (int otherId = 1; otherId <= parties; otherId++) {
       if (myId != otherId) {
-        DHParameterSpec dhSpec = DhParameters.getStaticDhParams();
-        Ot ot = new NaorPinkasOt(otherId, drbg, network, dhSpec);
+        AbstractNaorPinkasOT ot = new BigIntNaorPinkas(otherId, drbg, network);
```

Comment:
BigIntNaorPinkas has the same functionality as the old NaorPinkasOt.
The name is just different to differentiate between different NaorPinkas implementations.